src/dato/lib/server.clj

```diff
@@ -96,18 +96,15 @@
             (assert (casync/put! tx-report-ch tx)
                     "Cant put transaction on tx-report-ch")))))))
 
-(defn broadcast-tx! [dato-server sessions tx-report]
+(defn broadcast-tx! [tal-state session-store tx-report]
   (def last-tx-report tx-report)
-  (def l-dato-server dato-server)
   (let [data        (incoming/outgoing-tx-report tx-report)
         _           (def last-tx-data data)
-        payload     (ss-msg :server/database-transacted data)
-        enc-payload (cdc/encode payload :transit)]
-    (doseq [[_ session] sessions]
-      (let [ch (get-in session [:live :ch])]
-        (async/send! ch enc-payload)))))
+        message     (ss-msg :server/database-transacted data)]
+    (doseq [ch-id (all-channel-ids session-store)]
+      (tal/queue-msg! tal-state ch-id message))))
```

Comment:
Does Talaria have any concept of broadcast, or is it just about shuffling messages (and all the complexity that comes with that) between two points on a single channel?